R/estimate_KM.R

```diff
@@ -17,9 +16,11 @@
 #' 
 #' @seealso \code{\link[survival]{survfit.formula} \link[survival]{survfitCI}}
 #' 
-#' @param data The name of the ADaM Basic Data Structure (BDS) for Time-to-Event analysis eg ADTTE. Rows in which AVAL or CNSR contain NA, are removed during analysis. 
-#' @param strata Character vector, representing the strata for Time-to-Event analysis eg TRT01P. When NULL, an overall analysis is performed.
+#' @param data Data.frame for Time-to-Event analysis. Rows in which the analysis variable (AVAL) or the sensor variable (CNSR) contain NA, are removed during analysis. 
+#' @param strata Character vector, representing the strata for Time-to-Event analysis. When NULL, an overall analysis is performed.
 #'   Default is NULL.
+#' @param AVAL Analysis value for Time-to-Event analysis. Default is "AVAL".
```

Comment:
Do we need to explain that CNSR = 1 is assumed censored?

Reply:
Yes, I think that would be good